src/otx/v2/adapters/torch/dataset.py

```diff
@@ -0,0 +1,210 @@
+        drop_last: bool = False,
+        sampler: Sampler | (Iterable | dict) | None = None,
+        persistent_workers: bool = False,
+        **kwargs,
+    ) -> TorchDataLoader:
+        r"""Torch Based Dataset.subset_dataloader.
+
+        Args:
+            subset (str): Enter an available subset of that dataset.
+            pipeline (Optional[Union[list, dict]], optional):
+                Dataset Pipeline. Defaults to None.
+            batch_size (Optional[int], optional): How many samples per batch to load. Defaults to None.
+            num_workers (Optional[int], optional): How many subprocesses to use for data loading.
+                ``0`` means that the data will be loaded in the main process. Defaults to None.
+            config (Optional[Union[str, dict]], optional): Path to configuration file or Config.
+                Defaults to None.
+            shuffle (bool, optional): Set to ``True`` to have the data reshuffled at every epoch. Defaults to True.
+            pin_memory (bool, optional): If ``True``, the data loader will copy Tensors
+                into device/CUDA pinned memory before returning them.  If your data elements
+                are a custom type, or your :attr:`collate_fn` returns a batch that is a custom type,
+                see the example below. Defaults to False.
+            drop_last (bool, optional): value for whether to drop the last data when the batch is not divided up.
+                Defaults to False.
+            sampler (Optional[Union[Sampler, Iterable, Dict]], optional): Defines the strategy to draw
+                samples from the dataset. Can be any ``Iterable`` with ``__len__``
+                implemented. If specified, :attr:`shuffle` must not be specified.. Defaults to None.
+            persistent_workers (bool, optional): If ``True``, the data loader will not shutdown
+                the worker processes after a dataset has been consumed once. This allows to
+                maintain the workers `Dataset` instances alive. Defaults to False.
+            **kwargs (Any): Additional arguments to pass to the DataLoader constructor.
+
+        Returns:
+            torch.utils.data.DataLoader: Returns a subset of dataLoader.
+        """
+        # Config Setting
+        _config: dict = {}
+        if isinstance(config, str):
+            set_tuple_constructor()
+            with Path(config).open() as f:
+                _config = yaml.safe_load(f)
+        elif config is not None:
+            _config = config
+
+        subset_pipeline = pipeline
+        if isinstance(subset_pipeline, dict):
+            subset_pipeline = subset_pipeline[subset]
+        subset_dataset = self._build_dataset(subset=subset, pipeline=subset_pipeline, config=_config)
+        if batch_size is None:
+            batch_size = _config.get("batch_size", 1)
+        if num_workers is None:
+            num_workers = _config.get("num_workers", 0)
+
+        # kwargs conflict
```

Comment:
What does this mean?

Reply:
This is an unnecessary comment. Cleared.